packages/playwright-test/src/matchers/toMatchSnapshot.ts

```diff
@@ -49,6 +49,20 @@ export function toMatchSnapshot(this: ReturnType<Expect['getState']>, received:
   if (options.threshold === undefined && projectThreshold !== undefined)
     options.threshold = projectThreshold;
 
+  const projectPixelDelta = testInfo.project.expect?.toMatchSnapshot?.pixelCount;
+  if (options.pixelCount === undefined && projectPixelDelta !== undefined)
+    options.pixelCount = projectPixelDelta;
+
+  if (options.pixelCount !== undefined && options.pixelCount < 0)
+    throw new Error('`pixelCount` option value must be non-negative integer');
+
+  const projectPixelPercent = testInfo.project.expect?.toMatchSnapshot?.pixelRatio;
```

Comment:
I wonder if we should replace all of this with `options = { ...testInfo.project.expect?.toMatchSnapshot, ...options }`.

Reply:
good point!